src/storage/store_view_state.ts

```diff
@@ -5,6 +5,18 @@ export class ViewStateStorage {
     static putViewState(value:string){
         // This function adds the key:value => VIEW_STATE_KEY:'ui state' to storage
         chrome.storage.session.set({ [VIEW_STATE_KEY]: value });
+
+        //updates the right click option visibility accordingly
+        if(value === 'file-transfer') {
```

Comment:
Should this logic also go into the remove view state function when the user logs out?

Reply:
sounds good then that's it